packages/roosterjs-editor-plugins/lib/plugins/TableResize/editors/TableEditor.ts

```diff
@@ -24,6 +24,10 @@ const enum TOP_OR_SIDE {
     top = 0,
     side = 1,
 }
+
+// Class name for Table edit feature divs
+export const TEF_CLASS_NAME = '_roosterjsTableEditFeature';
```

Comment:
Please add comment like below, to avoid creating js document for this internal export:
/**
 * @internal
 */